packages/automerge-repo-network-messagechannel/package.json

```diff
@@ -13,7 +13,9 @@
     "test": "vitest"
   },
   "dependencies": {
-    "@automerge/automerge-repo": "^1.1.0-alpha.2"
+    "@automerge/automerge-repo": "workspace:*",
+    "debug": "^4.3.4",
+    "eventemitter3": "^5.0.1"
```

Comment:
pnpm is stricter about packages only using dependencies that they declare, whereas yarn let us get away with being sloppy.